docs/installation/docker.md

```diff
@@ -62,16 +61,20 @@ well.
    - You do not need "real" keys for basic development. Some features require
      certain keys, so you may be able to add them as you go.
 
-## Running the Docker app (basic)
+## Running the Docker app via docker-compose (recommended)
```

Comment:
Slightly more explicit in that it (a) mentions `docker-compose` and (b)  says "recommended" instead of "basic", as it's the workflow I work on/test.

Reply:
Related comment (cc @benhalpern): I think the `docker-run.sh` script might be quite outdated at this point, at least I never worked on it, so it lacks Elasticsearch, Sidekiq and other things. I think we need to make a call on whether or not we want to support two workflows (in which case I have to update the script) or not (in which case I remove the script).

Reply:
+1 on having only a recommended workflow

Reply:
I think we can remove the script. One workflow is great.